feature/users/data/src/commonMain/kotlin/io/timemates/app/users/data/CachedUsersDataSource.kt

```diff
@@ -0,0 +1,69 @@
+package io.timemates.app.users.data
+
+import io.timemates.app.users.data.database.CachedUsersQueries
+import io.timemates.sdk.common.constructor.createOrNull
+import io.timemates.sdk.files.types.value.FileId
+import io.timemates.sdk.users.profile.types.User
+import io.timemates.sdk.users.profile.types.value.EmailAddress
+import io.timemates.sdk.users.profile.types.value.UserDescription
+import io.timemates.sdk.users.profile.types.value.UserId
+import io.timemates.sdk.users.profile.types.value.UserName
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class CachedUsersDataSource(
+    private val cachedUsersQueries: CachedUsersQueries,
+    private val coroutineScope: CoroutineScope
+) {
+    init {
+        // TODO: Move it to other place
+        coroutineScope.launch { clear() }
+    }
+
+    private suspend fun clear() = withContext(Dispatchers.IO) {
```

Comment:
I think we still should clear cache by time, not by creating it. User can close the app and reopen it much faster. I think cache should serve ~10 minutes.